docs/features/annotations.md

```diff
@@ -12,9 +12,9 @@ Annotations is a feature introduced as part of the Speedgrader update to Autolab
 Hover over any line of the code and click on the green arrow, and the annotation form will appear. Add the comment, adjust the score, and select the targeted problem.
 
 When the "Add to shared comment pool" option is checked, the comment will be saved to a shared comment pool.
-When creating new annotations, shared comments matching what you have typed will be available via a dropdown list.
+When creating new annotations, shared comments matching what you have typed will be available via a dropdown list. Only the 50 most recently added shared comments will be displayed.
```

Comment:
haha good to specify how many shared comments will be displayed, 50 is quite a lot though so hopefully most people will never need to remember